l1-contracts/test/portals/UniswapPortal.sol

```diff
@@ -103,7 +103,7 @@ contract UniswapPortal {
 
     // Consume the message from the outbox
     {
-      IOutbox outbox = registry.getOutbox();
+      IOutbox outbox = registry.getRollup().OUTBOX();
```

Comment:
hmm, this adds the gas cost of another external call? will we end up going back on this change / adding a method to the registry anyway when golfing?

Reply:
It needs to be fixed as part of #7959.